apps/store/controllers/assets-router.jag

```diff
@@ -139,9 +141,20 @@ var validateType = function (type, tenantId){
  * @return {[type]}         Tenant id for the request
  */
 var getTenantId = function(request, session) {
+    var server = require('carbon').server;
     var tenantAPI = require('/modules/tenant-api.js').api;
-    var tenantDetails = tenantAPI.createTenantDetails(request, session);
-    var tenantId = tenantDetails.tenantId;
+    var tenantId;
+    var domain = request.getHeader("domain");
+    if(!domain){
```

Comment:
Thanks @daneshk , I have fixed it in [this commit](https://github.com/wso2/carbon-store/pull/842/commits/d825d8c011cf7d57ddf865fadb74847d5f3fa9fd)